### PR TITLE
WIP : Update contactus.php

### PR DESCRIPTION
**Pull request by nethu99**

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
Please drop the screenshot here that you are implemented

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
We need to implement extract UI, just like the Figma file.

**Comment by supuna97** [comment]
Try to write CSS in another class. Don't write inline CSS. Its not good for the SEO. @nethu99

File: components/contactus.php [hunk 1]
```diff
@@ -18,7 +19,7 @@
                 </div><br>
                 <div class="col-md-6">
                     <div class="form-group">
-                        <textarea name="txtemsg" class="form-control" placeholder="message" style="width: 100%; height: 150px;"></textarea>
+                        <textarea name="txtemsg" class="form-control" placeholder="message" style="width: 100%; height: 100px;"></textarea>
```
Comment (supuna97) [inline components/contactus.php:22]: Try to write CSS in another class. Don't write inline CSS. Its not good for the SEO. @nethu99

**Review by supuna97 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**